File: CourierAmexAPI/CourierAmex/Models/PackageNotesModel.cs
namespace CourierAmex.Models
{
    public class PackageNotesModel: BaseEntityModel<int>
    {
        public string? Codigo { get; set; }
        public string? Courier { get; set; }
        public string? Message { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime DueDate { get; set; }
        public string? IdUser { get; set; }
        public bool Sincronized { get; set; } = false;
        public string? CompanyName { get; set; } = null;
        public string? NombreCompleto { get; set; } = null;
        public string? Compannia { get; set; } = null;
        public int CompanyId { get; set; }
    }
}

File: CourierAmexClient/src/app/features/package/models/package-notes.model.ts
import { BaseEntity } from "@app/models";

//Tabla: CLI_NOTA
export interface PackageNotesModel extends BaseEntity<number> { 
    codigo: string;
    courier: string;
    message: string;
    createdAt: Date;
    dueDate: Date;
    idUser: string;
    sincronized: boolean;
    companyName: string;
    compannia: string;
    nombreCompleto: string;
    companyId: number;
}

export const newPackageNotes = {
    id: 0,
    status: 0,
    codigo: '',
    courier: '',
    message: '',
    createdAt: new Date(),
    dueDate: new Date(),
    idUser: '',
    sincronized: false,
    companyName: '',
    compannia:'',
    nombreCompleto:'',
    companyId:0
};